extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -237,6 +259,48 @@ export const ScamAssetWarning = ({
     }
   }, [warningRef]);
 
+  const handleSubmit = async () => {
+    setIsSubmitting(true);
+
+    const server = new StellarSdk.Server(networkDetails.networkUrl);
+    const sourceAccount: Account = await server.loadAccount(publicKey);
+    const transactionXDR = new StellarSdk.TransactionBuilder(sourceAccount, {
+      fee: xlmToStroop(recommendedFee).toFixed(),
+      networkPassphrase: networkDetails.networkPassphrase,
+    })
+      .addOperation(
+        StellarSdk.Operation.changeTrust({
+          asset: new StellarSdk.Asset(code, issuer),
+        }),
+      )
+      .setTimeout(180)
+      .build()
+      .toXDR();
+
+    const res = await dispatch(
+      signFreighterTransaction({
+        transactionXDR,
+        network: networkDetails.networkPassphrase,
+      }),
+    );
+
+    if (signFreighterTransaction.fulfilled.match(res)) {
+      const submitResp = await dispatch(
+        submitFreighterTransaction({
+          signedXDR: res.payload.signedTransaction,
+          networkDetails,
+        }),
+      );
+      if (submitFreighterTransaction.fulfilled.match(submitResp)) {
+        navigateTo(ROUTES.account);
+        emitMetric(METRIC_NAMES.manageAssetAddAsset, { code, issuer });
```

Comment:
pretty minor - but wondering if we should use a new METRIC_NAMES here to differentiate from adding a trustline that wasn't flagged

Reply:
ah ya  I like that idea, for better tracking 
https://github.com/stellar/freighter/pull/631/commits/e7f9f422dc4dbc8eede89d940274cc2f06430f01